sw/airborne/modules/computer_vision/colorfilter.c

```diff
@@ -58,6 +68,17 @@ struct image_t *colorfilter_func(struct image_t *img)
                                        color_cr_min, color_cr_max
                                       );
 
+
+  if (COLORFILTER_SEND_OBSTACLE) {
+    if (color_count > 20)
+      AbiSendMsgOBSTACLE_DETECTION(ABI_BROADCAST, 1,
```

Comment:
You are not supposed to send a message with  `ABI_BROADCAST` id, this is only for receiving. You should add the ID of your module in `subsystems/abi_sender_ids.h` (choose a "category" matching your module or start a new one) and use it here.